local-cli/runAndroid/runAndroid.js

```diff
@@ -150,9 +152,9 @@ function tryInstallAppOnDevice(args, device) {
   }
 }
 
-function tryLaunchAppOnDevice(device, packageName, adbPath, mainActivity) {
+function tryLaunchAppOnDevice(device, packageNameWithSuffix, packageName, adbPath, mainActivity) {
   try {
-    const adbArgs = ['-s', device, 'shell', 'am', 'start', '-n', packageName + '/.' + mainActivity];
+    const adbArgs = ['-s', device, 'shell', 'am', 'start', '-n', packageNameWithSuffix + '/' + packageName +  + mainActivity];
```

Comment:
@Phredward Yeah. I saw that. I won't be able to make any changes to this PR. If you are unable to do it, I can do it in a separate PR but I would rather the authorship stay with you for working on this :)